packages/functional-tests/tests/subscription.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { test, expect } from '../lib/fixtures/standard';
+
+test.describe('severity-1', () => {
+  test('subscribe with credit card and login to product', async ({
```

Comment:
nit: we say "credit card" here. should we clarify to "stripe credit card"?
Not sure it matters, or if we ever want to grep by "stripe" to run specific tests.

Reply:
good suggestion, not sure if it's required but good to have. I will add it.